README.md

```diff
@@ -19,34 +19,101 @@ In addition to this README file, there are docs covering the following topics:
 * [Design](docs/design.md)
   * Describes the technical design and implementation details of the Version Control app
 
-## Local Dev & Test Environment
+## Installation
+
+### Installation Considerations
+
+There are some special considerations for running the Version Control app:
+
+* Nautobot 1.1.0 or later is required
```

Comment:
```suggestion
* Nautobot 1.2.0 or later is required
```

I guess technically it could work with 1.1.0, but the "Current Branch" banner is now exclusive to 1.2 and later, and the PluginConfig for this plugin sets a minimum Nautobot version of 1.2.0-beta.1 at present.

Reply:
Ah yes thanks.